source/compiler/sc2.c

```diff
@@ -2133,10 +2133,10 @@ SC_FUNC void lexinit(void)
   _lexnewline=FALSE;
 }
 
-char *sc_tokens[] = {
+char *sc_tokens[tlexLAST-tFIRST+1] = {
```

Comment:
This seems to be the only place `tlexLAST` is used.  It wasn't in the original, doesn't fit with the naming scheme of the other meta-tokens, could be confused with `tLAST`, and isn't needed to define the array size.  So why it is used?

Reply:
It was a futile attempt to link `sc_tokens` with the token list.

I'll remove it.